src/main/java/frc/robot/commands/GoHome.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.arm.Arm.ArmPosition;
+import frc.robot.subsystems.claw.Claw;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.elevator.Elevator.ElevatorHeight;
+import frc.robot.subsystems.climb.Climb;
+import frc.robot.subsystems.climb.Climb.ClimbPosition;
+import frc.robot.subsystems.lights.Lights.LightCode;
+import frc.robot.subsystems.lights.Lights;
+
+public class GoHome extends SequentialCommandGroup {
+    public GoHome(Climb climb, Elevator elevator, Arm arm, Claw claw, Lights lights) {
+      addRequirements(climb, elevator, arm, claw);
+      addCommands(
+        new InstantCommand(() -> climb.setDesiredClimbPosition(ClimbPosition.STOWED)),
+        new InstantCommand(() -> arm.setDesiredPosition(ArmPosition.HOME)),
```

Comment:
Do a conditional command
if arm in interference zone, move arm to HOME, wait until reaches pos,  then climb to STOWED, wait
if not, then move climb to STOWED, wait until reaches pos, then move arm to HOME, wait